src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -564,6 +642,42 @@ private async Task LoadDataAsync()
             }
         }
 
+        private bool IsVaultFilterMyVault => 
+            _vaultFilterSelection != null && _vaultFilterSelection == AppResources.MyVault;
```

Comment:
`_vaultFilterSelection != null` is not needed given that `null` is different than `AppResources.MyVault`, you can write it directly as `private bool IsVaultFilterMyVault => _vaultFilterSelection == AppResources.MyVault;`